src/Responsive.php

```diff
@@ -68,6 +68,10 @@ public static function getPresets(Asset $asset): array
         $index = 0;
 
         foreach ($configPresets as $preset => $data) {
+            if(!($data['w'] ?? false)) {
```

Comment:
If `w` is `null` you shouldn't have to do the `?? false`.  Also; shouldn't we also have the width config? Maybe in the future add a validator that validates the config?